SPECS/reaper/reaper.spec

```diff
@@ -35,6 +35,7 @@ Source7:        reaper-local-n-%{version}-%{local_n_release}.tar.gz
 Patch0:         CVE-2024-37890.patch
 Patch1:         CVE-2023-42282.patch
 Patch2:         CVE-2017-18214.patch
+Patch3:         DER-signature-decoding-correction.patch
```

Comment:
We need to rename the patch file to CVE-2024-42459 and then add 2 .nopatch files with explanations about how the former fixes the other two issues.

Tooling is a little dumb when it comes to aggregating more than 1 CVE into a single PR.